src/app/services/date.service.ts

```diff
@@ -0,0 +1,15 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class DateService {
+
+  constructor() { }
+
+  minutesToMiliseconds(minutes: number) {
+    const miliseconds = minutes * 60000;
```

Comment:
what is `60000`? This is the most important part of this exercise, I cant understand what is `60000`. To avoid my doubts, what if you add this value into a variable with a good name/context? I will suggest:

`const MINUTE_IN_MILLISECONDS = 60000`

it would be perfect if you declare this variable as a readonly class variable, so it would be reusable.